dxr/plugins/xpidl/indexers.py

```diff
@@ -0,0 +1,45 @@
+from os.path import join
+
+from xpidl.xpidl import IDLParser, IDLError
+
+import dxr.indexers
+from dxr.indexers import iterable_per_line, with_start_and_end, split_into_lines
+from dxr.plugins.xpidl.filters import PLUGIN_NAME
+from dxr.plugins.xpidl.visitor import IdlVisitor
+
+
+class FileToIndex(dxr.indexers.FileToIndex):
+    def __init__(self, path, contents, plugin_name, tree):
+        super(FileToIndex, self).__init__(path, contents, plugin_name, tree)
+        self.temp_folder = join(self.tree.temp_folder, 'plugins', PLUGIN_NAME)
+        self.parser = IDLParser(self.temp_folder)
+        self._idl = None
+        self._idl_exception = False
```

Comment:
I expected this to be an actual exception based on the name. Maybe `_had_idl_exception`?